src/OctoshiftCLI.Tests/gei/Commands/GenerateScriptCommandTests.cs

```diff
@@ -620,6 +621,22 @@ public async Task It_Uses_Ado_Pat_When_Provided()
             mockEnvironmentVariableProvider.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public void It_Uses_Skip_Releases_When_Provided()
```

Comment:
just a nit rename recommendation:
```suggestion
        public void It_Adds_Skip_Releases_To_Migrate_Repo_Command_When_Provided()
```

Reply:
Updated in 5410d763c4be7c97221ebf2828edcda6d2326a56!